go.mod

```diff
@@ -2,7 +2,4 @@ module cockroach-db-cursor-attempt
 
 go 1.21
 
-require (
-	github.com/google/uuid v1.6.0
-	github.com/stretchr/testify v1.8.4
```

Comment:
Not sure why these dependencies were included in the first place.

TODO: find purpose of testify module